### DISCUSSION TITLE
Is it possible for Doplarr to notify the channel when a request has been submitted?

### DISCUSSION THREAD

**Discussion by kevincam** [activexray/Doplarr#120] [Q&A]
Right now Doplarr does all the messaging in ephemeral messages, which is great for the majority of the process. But it would be great to get a channel message just for the actual add step, so it's easy to see who requested what. This also makes up for a gap in Sonarr & Radarr's notification system where they don't notify when something is initial added.

**Comment by activexray** [comment]
Yould set the `:discord/requested-msg-style` variable to either `:plain` or `:embed`

**Comment by kevincam** [comment]
I've adjusted this setting back and forth and added every permission in the messages section to the bot oauth request but it still doesn't post any public messages to the channel. I also don't see any errors in the console during request. Is there any additional configuration required?
